docs/external-configuration-parameter-values.md

```diff
@@ -0,0 +1,61 @@
+---
+id: external-configuration-parameter-values
+title: External configuration parameter values
+---
+
+An external HTTP endpoint of yours can provide the possible values of a
+configuration parameter.
+
+Add on the product configuration page, on the configuration parameters tab,
+the full URL of the endpoint that implements the spec described here.
+
+## Request
+
+Your endpoint will receive a POST request with a JSON body content like this:
+
+```json
+{
+  "productId": 123,
+  "productIdentifier": "your-product",
+  "productVersionId": 456,
+  "productVersionIdentifier": "your-product-plan",
+  "configurationParameter": "CUSTOM_DOMAIN",
+  "language": "en"
```

Comment:
cloudCredential? if set

Reply:
maybe also cloud provider could be useful

Reply:
in another PR, it's missing from the order validation payload too